client/src/components/layout/sidebar/sidebar.tsx

```diff
@@ -0,0 +1,32 @@
+import SidebarListItem from "./sidebar-list-item";
+import { CharacterWink, InfoIcon } from "@/components/common/icon";
+
+interface SidebarProps {
+  version: string;
+  onLogout: () => void;
+}
+
+export const Sidebar = ({ version, onLogout }: SidebarProps) => {
+  return (
+    <div className="flex flex-col justify-between items-center flex-shrink-0 w-[280px] h-[calc(100vh_-_56px)] py-10">
```

Comment:
提案を適用する場合はこちらも修正お願いします🙏

```suggestion
    <div className="flex flex-col justify-between items-center w-[280px] h-full bg-white py-10 shadow-lg">
```

Reply:
https://github.com/user-attachments/assets/90b191c7-73a2-4147-b6de-a6a7c803fe2e

これらを適用するとこんな感じ